chrome-extension/src/index.js

```diff
@@ -1,19 +1,24 @@
 let editor;
 let iframe;
+const isProduction = true;
```

Comment:
function isDevMode() {
    return !('update_url' in chrome.runtime.getManifest());
}